lib/weka/core/instances.rb

```diff
@@ -49,12 +49,16 @@ def instances
         enumerate_instances.to_a
       end
 
-      def attributes
-        enumerate_attributes.to_a
+      def attributes(include_class_attribute = false)
+        attrs = enumerate_attributes.to_a
+        if include_class_attribute && class_attribute_defined?
+          attrs.insert(class_index, class_attribute)
+        end
+        attrs
```

Comment:
Could you please add some spaces between the `if`-block and the single lines (also applies for map/inject blocks). It eases reading the code a lot. :)